exotic.py

```diff
@@ -523,6 +540,25 @@ def plate_solution(fits_file, saveDirectory):
             return False
         time.sleep(5)
 
+#Checks if comparison star coordinates don't point to variable stars
+def variableStarCheck(comparisonStarFile):
+    #Open WCS file and convert comparison star's coordinates into normal coordinates
+    starList = fits.open(comparisonStarFile)
+    wcsCoords = WCS.WCS(starList[0].header)
+    starCoords = wcsCoords.all_pix2world()
+
+    #Use normalized Coordinates to search on SIMBAD to check if that star is variable
+    try:
+        resultTable = Simbad.query_region(coord.SkyCoord(starCoords[0], starCoords[1], unit = (u.deg, u.deg))
```

Comment:
I don't see you looping over different stars in the image or passing the RA+DEC (or pixel location) of the comparison star. Looks like you pass the fits file to your function, open the fits file, load in the WCS coordinates, convert the pixel values to RAs and DECs, and then read in `starCoords[0]` and `starCoords[1]`, which are all (?) the RAs and DECs in the file. Or am I missing something?

Reply:
Please see updated code